Looks real:
```
======================================================================
FAIL: test_location_update (corehq.apps.users.tests.bulk_upload.TestUserBulkUpload)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/users/tests/bulk_upload.py", line 142, in test_location_update
    self.assertEqual(self.user.location_id, self.location._id)
AssertionError: None != u'7eca110c0d0228fd39c2e3b91b511ee9'
```

--- a/corehq/apps/users/tests/bulk_upload.py
+++ b/corehq/apps/users/tests/bulk_upload.py
@@ -107,7 +107,6 @@ class TestUserBulkUpload(TestCase, DomainSubscriptionMixin):
         self.domain.delete()
 
     @property
-    @memoized
     def user(self):
         return CommCareUser.get_by_username('{}@{}.commcarehq.org'.format(
             self.user_specs[0]['username'],

--- a/corehq/apps/users/tests/bulk_upload.py
+++ b/corehq/apps/users/tests/bulk_upload.py
@@ -124,6 +124,7 @@ class TestUserBulkUpload(TestCase, DomainSubscriptionMixin):
         self.assertEqual(self.user_specs[0]['phone-number'], self.user.phone_number)
         self.assertEqual(self.user_specs[0]['name'], self.user.name)
 
+    @patch('corehq.apps.users.bulkupload.domain_has_privilege', lambda x, y: True)
     def test_location_update(self):
         self.setup_location()
         from copy import deepcopy
@@ -138,10 +139,8 @@ class TestUserBulkUpload(TestCase, DomainSubscriptionMixin):
         )
         self.assertEqual(self.user.location_id, self.location._id)
         self.assertEqual(self.user.location_id, self.user.user_data.get('commcare_location_id'))
-        self.teardown_subscription()
 
     def setup_location(self):
-        self.setup_subscription(self.domain_name, SoftwarePlanEdition.ADVANCED)
         self.state_code = 'my_state'
         self.location = make_loc(self.state_code, type='state', domain=self.domain_name)
 

--- a/corehq/apps/users/tests/bulk_upload.py
+++ b/corehq/apps/users/tests/bulk_upload.py
@@ -88,12 +88,11 @@ class UserLocMapTest(CommTrackTest):
 
 class TestUserBulkUpload(TestCase, DomainSubscriptionMixin):
 
-    @classmethod
-    def setUpClass(cls):
-        cls.domain_name = 'mydomain'
-        cls.domain = Domain(name=cls.domain_name)
-        cls.domain.save()
-        cls.user_specs = [{
+    def setUp(self):
+        self.domain_name = 'mydomain'
+        self.domain = Domain(name=self.domain_name)
+        self.domain.save()
+        self.user_specs = [{
             u'username': u'hello',
             u'user_id': u'should not update',
             u'name': u'Another One',
@@ -104,9 +103,8 @@ class TestUserBulkUpload(TestCase, DomainSubscriptionMixin):
             u'email': None
         }]
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.domain.delete()
+    def tearDown(self):
+        self.domain.delete()
 
     @property
     @memoized

